test/ConfigDiscoveryTest.php

```diff
@@ -87,24 +95,24 @@ public function createModulesConfig()
     public function assertOptionsContainsNoopInjector(Collection $options)
     {
         if ($options->isEmpty()) {
-            throw new ExpectationFailedException('Options array is empty; no NoopInjector found!');
+            $this->fail('Options array is empty; no NoopInjector found!');
```

Comment:
Not sure if it is gonna work with PHPUnit 6.0:
https://github.com/sebastianbergmann/phpunit/blob/master/src/Framework/ExpectationFailedException.php#L21